blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserImpl.java

```diff
@@ -36,15 +47,17 @@
 
     protected final User user;
 
+    private AccessControlled orgBase;
+
     private final Reachable parent;
     public UserImpl(User user, Reachable parent) {
```

Comment:
UserImpl and UserContainer should take in a BlueOrganization. Then you can delete the from URL code.